src/hotspot/share/opto/addnode.cpp

```diff
@@ -627,10 +626,9 @@ Node *AddPNode::Ideal(PhaseGVN *phase, bool can_reshape) {
     if( t22->singleton() && (t22 != Type::TOP) ) {  // Right input is an add of a constant?
       set_req(Address, phase->transform(new AddPNode(in(Base),in(Address),add->in(1))));
       set_req(Offset, add->in(2));
-      PhaseIterGVN *igvn = phase->is_IterGVN();
-      if (add->outcnt() == 0 && igvn) {
+      if (add->outcnt() == 0 && phase->is_IterGVN()) {
```

Comment:
Again, I think replacing `igvn` by `phase->is_IterGVN()` makes the code less readable.